catroid/src/main/res/values/strings.xml

```diff
@@ -1889,7 +1889,8 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_logic_comparison">Comparison operators</string>
     <string name="formula_editor_object_general">General properties</string>
     <string name="formula_editor_object_movement">Motion properties</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Face sensors</string>
+    <string name="formula_editor_device_text_detection">Text sensors</string>
```

Comment:
Please instead use "Face detection" and "Text recognition"

Reply:
Updated.